hosts_example.yml

```diff
@@ -165,13 +165,31 @@ kafka_broker:
       ## By default the first host will get broker id=1, second gets id=2. Set broker_id to customize
       # broker_id: 3
 
+      ## Properties can also be applied on a host by host basis.
+      # In the below example we configure a Multi-Region Replication by setting the following properties on each host:
+      # * broker.rack
```

Comment:
I'd prefer this to take less lines. Also the NOTE is unnecessary, ppl can (and should) read the MRC docs themselves